### DISCUSSION TITLE
What folders are needed for chaindata snapshot and backup

### DISCUSSION THREAD

**Discussion by miohtama** [erigontech/erigon#6891] [Q&A] [answered]
After three weeks of syncing, I have finally managed to sync BNB Chain on Erigon node.

I'd like to take a snapshot of data files, so I do not need to sync from the scratch again. What folders I would need to include in the backup. I see:

```
drwxr--r-- 2 1000 1000  50 Jan 25 12:38 bor
drwxr--r-- 2 1000 1000  50 Jan 25 12:38 chaindata
-rw------- 1 1000 1000  66 Jan 25 12:38 jwt.hex
-rw------- 1 1000 1000   0 Jan 25 12:38 LOCK
-rw------- 1 1000 1000  64 Jan 25 12:38 nodekey
drwxr--r-- 3 1000 1000  27 Jan 25 12:38 nodes
drwxr--r-- 2 1000 1000  50 Jan 25 12:38 parlia
drwxr--r-- 4 1000 1000 32K Feb 17 09:34 snapshots
drwxr-xr-x 2 1000 1000  10 Feb 17 09:34 temp
```

I assume for the BNB Chain I need to copy only `chaindata`, `nodes`, `parlia`. 

If I restore these files on another server Erigon will resume correctly from the last block that was before backup?

(Not sure if `bor` is supposed to be there, as I thought Bor was Polygon thing - maybe a leftover from other chain sync.)

**Comment by AskAlexSharov** [comment, marked as answer]
And “snapshots” dir
https://github.com/ledgerwatch/erigon#datadir-structure
